qiskit_experiments/framework/experiment_data.py

```diff
@@ -188,13 +190,23 @@ def __init__(
             backend: Backend the experiment runs on. This overrides the
                 backend in the experiment object.
             service: The service that stores the experiment results to the database
+            provider: The provider used for the experiments
+                (can be used to automatically obtain the service)
             parent_id: ID of the parent experiment data
                 in the setting of a composite experiment
             job_ids: IDs of jobs submitted for the experiment.
             child_data: List of child experiment data.
             verbose: Whether to print messages.
             db_data: A prepared ExperimentDataclass of the experiment info.
                 This overrides other db parameters.
+
+        Additional info:
+            In order for to save the experiment data to the resultDB, the class
```

Comment:
ResultsDB shouldn't be in the docstring. Use something like cloud service instead.